table/tables/index_test.go

```diff
@@ -39,33 +39,25 @@ import (
 	"github.com/pingcap/tidb/util/collate"
 	"github.com/pingcap/tidb/util/mock"
 	"github.com/pingcap/tidb/util/rowcodec"
-	"github.com/pingcap/tidb/util/testleak"
+	"github.com/stretchr/testify/require"
 )
 
-var _ = Suite(&testIndexSuite{})
-
 type testIndexSuite struct {
 	s   kv.Storage
 	dom *domain.Domain
 }
 
-func (s *testIndexSuite) SetUpSuite(c *C) {
-	testleak.BeforeTest()
+var s = testIndexSuite{}
+
+func TestSetUpSuite(t *testing.T) {
```

Comment:
setup should not be a test, you should batch setup logic and use when necessary

Reply:
what is `batch setup logic ` mean?

Reply:
ok, thanks.